src/processTargets/modifiers/OrdinalRangeSubTokenStage.ts

```diff
@@ -37,17 +37,12 @@ export default class OrdinalRangeSubTokenStage implements ModifierStage {
       throw new Error("Subtoken exclusions unsupported");
     }
 
-    if (this.modifier.scopeType.type === "word") {
-      pieces = [...tokenText.matchAll(SUBWORD_MATCHER)].map((match) => ({
-        start: match.index!,
-        end: match.index! + match[0].length,
-      }));
-    } else if (this.modifier.scopeType.type === "character") {
-      pieces = range(tokenText.length).map((index) => ({
-        start: index,
-        end: index + 1,
-      }));
-    }
+    const regex =
+      this.modifier.scopeType.type === "word" ? SUBWORD_MATCHER : SPLIT_REGEX;
+    pieces = [...tokenText.matchAll(regex)].map((match) => ({
+      start: match.index!,
+      end: match.index! + match[0].length,
+    }));
```

Comment:
This seemed worth fixing while we were here; we weren't properly handling combining chars.  @AndreasArvidsson you'll be happy to see we've replaced another scope type by a regex 😄

Reply:
Nice :D